### DISCUSSION TITLE
Shamsi calendar format

### DISCUSSION THREAD

**Discussion by BehzadDara** [NG-ZORRO/ng-zorro-antd#8408] [General] [answered]
I want to use the shamsi calendar format with the date picker component from ng-zorro. However, when I set the locale to fa, it only changes the language to farsi, not the calendar. How can I make the date picker use the shamsi calendar in my project?

**Comment by ParsaArvanehPA** [comment, marked as answer]
Greetings @BehzadDara,

Have you considered using [this package](https://www.npmjs.com/package/ng-met-antd)?
It is a good solution, but it has a limitation: it does not support angular version 17 yet.
